kingdom-of-ocelader/TrestlebridgeFarms
# Cm fixing grazing

# Description

Purchasing animals now displays all animals you can buy and works as expected.

Applies to issue # ()

## Type of change

Please delete options that are not relevant.



# Testing Instructions

Run application
Go to purchase animals
all animals should appear and be able to be bought



# Checklist:
